libs/design-system/src/lib/Components/LandingHeader/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ButtonLink } from '../Button';
+import { GradientName, LandingHeaderContainer } from './LandingHeaderContainer';
+
+export type LandingHeaderProps = {
+  title: string;
+  gradient: GradientName;
+  callout: string;
+  description: string;
+  buttonText: string;
+  buttonUrl: string;
+};
+
+export function LandingHeader({
+  buttonText,
+  buttonUrl,
+  callout,
+  description,
+  gradient,
+  title,
+}: LandingHeaderProps) {
+  return (
+    <LandingHeaderContainer gradient={gradient}>
+      <div className="flex max-w-6xl flex-col justify-items-stretch px-4 py-12 md:px-20 md:py-20">
+        <h1 className="text-h1 mb-6 max-w-full overflow-hidden text-ellipsis !text-4xl md:!text-7xl">
```

Comment:
Good call on adjusting these font sizes. I think we can just update the text-h1, we'll probably run into those overflow issues in other places. The mb can be 12 here